src/badguy/dispenser.cpp

```diff
@@ -147,20 +149,21 @@ Dispenser::Dispenser(const ReaderMapping& reader) :
 void
 Dispenser::draw(DrawingContext& context)
 {
-  if (m_type != DispenserType::POINT || Editor::is_active()) {
+  if (m_type != DispenserType::POINT || Editor::is_active())
     BadGuy::draw(context);
-  }
 }
 
 void
 Dispenser::activate()
 {
-  if (m_broken){
+  if (m_broken)
     return;
-  }
-  if (m_autotarget && !m_swivel){ // auto cannon sprite might be wrong
+
+  if (m_autotarget && !m_swivel)
+  { // auto cannon sprite might be wrong
     auto* player = get_nearest_player();
```

Comment:
Can this comment be on a line of its own? It's a small thing but it tickles me :sweat_smile: